File: 9-palindrome-number/9-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        
        String _check = String.valueOf(x);
        for (int i = 0; i < _check.length()/2; i++) {
            if (_check.charAt(i) == _check.charAt(_check.length()-1-i)) {
                continue;
            } else {
                return false;
            }
        }
        return true;
    }
}

File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string number = to_string(x);
        for (int i=0;i<number.length()/2;i++) {
            if (number.at(i)==number.at(number.length()-1-i)) {
                continue;
            } else return false;
        }
        return true;
    }
};